app/main/page.tsx

```diff
@@ -33,8 +34,15 @@ const defaultWaveHandButtons = [
     'Support',
 ];
 
+const defaultAffirmations = [
+    'Say what I want to say, whatever happens will help me grow',
+    'I can take up space',
+    'I have an important voice',
+    'Feel the tension and proceed',
+    'I have the right to stutter',
+]
+
 function App() {
- 
   const { state, 
   setCurrentAffirmation,
   setAllAffirmations,
```

Comment:
Shouldn't adding the AffirmationCarousel mean that the affirmations don't need to be saved in the state object anymore?

Reply:
It's still used by the affirmation tab (line 90-92). I will remove this state after refactoring that tab.